backend/core/src/listings/listings.service.ts

```diff
@@ -143,7 +146,33 @@ export class ListingsService {
           : listing.closedAt,
     })
 
-    return await this.listingRepository.save(listing)
+    const saveResponse = await this.listingRepository.save(listing)
+    /**
+     * Send purge request to Nginx.
+     * Wrapped in try catch, because it's possible that content may not be cached in between edits,
+     * and will return a 404, which is expected.
+     * listings* purges all /listings locations (with args, details), so if we decide to clear on certain locations,
+     * like all lists and only the edited listing, then we can do that here (with a corresponding update to nginx config)
+     */
+    if (process.env.PROXY_URL) {
+      firstValueFrom(
```

Comment:
I think we need to await the `firstValueFrom()` calls or there is a chance that an error is swallowed with the function already having been returned

Reply:
Good point. I chose not to do an await as I thought the calls shouldn't be a blocker to the main request. But that makes sense, especially since we are still catching the errors so it shouldn't block the overall call to the update endpoint

Reply:
@YazeedLoonat this has been added